monero-harness/rustfmt.toml

```diff
@@ -1,7 +1,7 @@
 edition = "2018"
 condense_wildcard_suffixes = true
 format_macro_matchers = true
-merge_imports = true
+imports_granularity = "Crate"
```

Comment:
You may enjoy the new `module` setting, it is much better than `crate`.

Reply:
As far as I could see I am happy with `"Crate"`

Reply:
Module has the advantage of causing a lot less merge conflicts because it breaks it up into multiple lines, thereby causing less whitespace change on adding/removing imports.

Reply:
Hum, we could consider it in a follow-up PR.